timbermill-local/src/main/java/com/datorama/oss/timbermill/ElasticsearchClient.java

```diff
@@ -86,6 +86,10 @@
 	private int maxBulkIndexFetches; // after such number of fetches, bulk is considered as failed and won't be persisted anymore
 	private LinkedBlockingQueue<Pair<DbBulkRequest, Integer>> failedRequests = new LinkedBlockingQueue<>(100000);
 	private DiskHandler diskHandler;
+	private int numOfBulksPersistedToDisk = 0;
+	private int numOfSuccessfullBulksFromDisk = 0;
+	private int numOfFetchedMaxTimes = 0;
+	private int numOfCouldntBeInserted = 0;
```

Comment:
all these members are only temporary - they will help to analyze the persistence performance